### PR TITLE
feat: Navbar component

### PR DESCRIPTION
**Pull request by Maikittitee**
feat: Navbar component

### CODE REVIEW

File: src/app/footer.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+// import { FaInstagram, FaFacebook, FaTiktok } from 'react-icons/fa'; // Icons from react-icons
+
+const Footer: React.FC = () => {
+  return (
+    <footer className="bg-gray-200 p-8 text-center">
+      <h2 className="mb-4 text-lg font-bold">ติดตามข่าวสาร</h2>
+      <div className="flex justify-center space-x-8">
+        {/* <FaInstagram size={40} /> */}
+        {/* <FaFacebook size={40} /> */}
+        {/* <FaTiktok size={40} /> */}
```
Comment (Kittonn) [inline src/app/footer.tsx:11]: use iconify icon
ref: https://www.npmjs.com/package/@iconify/react

File: src/app/navbar.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+const NavBar: React.FC = () => {
+  return (
+    <nav className="flex items-center justify-between bg-gray-200 p-4">
+      <h1>CE Smart Career 2024</h1>
+      <ul className="flex space-x-6 font-bold">
+        <li>เกี่ยวกับ</li>
+        <li>สัมมนา</li>
+        <li>บริษัท</li>
+        <li>สถานที่</li>
+        <li>ช่องทางติดต่อ</li>
```
Comment (Kittonn) [inline src/app/navbar.tsx:12]: ทำเป็น constants แล้วลูปเอา

File: package-lock.json
Comment (Kittonn) [inline package-lock.json:1]: remove this 
we use pnpm

File: src/app/footer.tsx (hunk 1 above)
Comment (Peqchji) [reply src/app/footer.tsx:11]: function Footer() pls
we all do this format

File: src/app/navbar.tsx (hunk 2 above)
Comment (Kittonn) [reply src/app/navbar.tsx:12]: responsive design

File: src/app/footer.tsx
Comment (Kittonn) [inline src/app/footer.tsx:1]: move to modules/home/_sections

File: src/app/navbar.tsx
Comment (Kittonn) [inline src/app/navbar.tsx:1]: move to shared/components

**Pull request merged**